readme.md

```diff
@@ -0,0 +1,67 @@
+# cvsclone
+
+Utility to clone cvs repositories over the cvspserver interface.
+
+As ```git cvsimport``` is pretty slow over the wire (and as [cvs2git](http://cvs2svn.tigris.org/cvs2git.html) can do a substantially better job if there are a lot of branches), it is often better to "clone" a whole CVS repository. Enter cvsclone!
+
+## Features
+
+- Works with anonymous access. Will read $HOME/.cvspass if needed
+- Can clone corrupt repositories: writes ,v files directly, does not need rcs (For example, ccvs module has archives that go backwards in time).
+
+## Building
+
+You will need [Flex](http://flex.sourceforge.net/) installed to create cvsclone. You can normally find this in your package manager i.e:

```

Comment:
I'd think "e.g." would be maybe a better fit here than "i.e.", no? I mean apt-get is I think only present on select distributions?